apps/console/src/components/pages/protected/evidence/evidence-table.tsx

```diff
@@ -31,6 +32,19 @@ const EvidenceTable = ({ evidences, control, canEdit }: Props) => {
     controlIdFromControl: control.controlID!,
     subcontrolIdFromControl: control.subcontrolID || undefined,
   }
+
+  const controlParam: CustomEvidenceControl = {
+    id: control.controlID || (control.subcontrolID as string),
+    referenceFramework: control.subcontrolReferenceFramework
+      ? Object.values(control.subcontrolReferenceFramework)[0] ?? ''
+      : control.referenceFramework
+      ? Object.values(control.referenceFramework)[0] ?? ''
+      : '',
+
+    refCode: control.displayID ?? '',
```

Comment:
Not sure if there is the only place but I noticed the task -> evidence creation now has display ID, this should _always_ be refCode for controls and subcontrols 
<img width="553" height="410" alt="image" src="https://github.com/user-attachments/assets/c3358ae4-2e99-4df0-804c-cdc65b269021" />

This appears to only be an issue on creating evidence tasks - so I'm a bit confused why the logic is changing on the section when we are only adding suggested controls and shouldn't be changing else - all evidence slideouts should be the same?

```suggestion
    refCode: control.refCode ?? '',
```